src/events/event.cpp

```diff
@@ -623,6 +659,16 @@ EventTimeOfInterestReached::EventTimeOfInterestReached(const Time* time_,
     , camera(camera_)
 {}
 
+EventMissionAdded::EventMissionAdded(const std::string_view name)
+    : Event(Type)
+    , name(temporaryString(name))
+{}
+
+EventMissionRemoved::EventMissionRemoved(const std::string_view name)
+    : Event(Type)
+    , name(temporaryString(name))
+{}
+
```

Comment:
```suggestion
EventMissionAdded::EventMissionAdded(const std::string_view name_)
    : Event(Type)
    , name(temporaryString(name_))
{}

EventMissionRemoved::EventMissionRemoved(const std::string_view name_)
    : Event(Type)
    , name(temporaryString(name_))
{}

```

Since the parameter to the constructor has the same name as the member